### PR TITLE
Fixing no parseable (by the i18n extractor) optional chaining construction

### PR DESCRIPTION
**Pull request by jespino**
#### Summary
Fixing no parseable (by the i18n extractor) optional chaining construction

### CODE REVIEW

File: app/components/post_body_additional_content/index.js [hunk 1]
```diff
@@ -52,11 +52,13 @@ function makeMapStateToProps() {
...
             getBool(state, Preferences.CATEGORY_DISPLAY_SETTINGS, Preferences.LINK_PREVIEW_DISPLAY, true);
 
+        const openGraphData = getOpenGraphMetadataForUrl(state, link) || getOpenGraphData(ownProps.metadata, link);
+
         return {
             ...getDimensions(state),
             googleDeveloperKey: config.GoogleDeveloperKey,
             link,
-            openGraphData: getOpenGraphMetadataForUrl(state, link) || getOpenGraphData(ownProps.metadata, link)?.data,
+            openGraphData: openGraphData?.data,
```
Comment (enahum) [inline app/components/post_body_additional_content/index.js]: This is wrong.
`data` Property is only part of the second option in the or statement.

**Comment by jespino** [comment]
Yes, lets do that.

**Comment by enahum** [comment]
And this one should be cherry picked too

**Comment by jespino** [comment]
I think now is following properly the previously existing logic.

**Pull request merged**